test/_modules/Context.t.sol

```diff
@@ -100,6 +100,14 @@ contract ContextTest is Test {
 
         target.value{value: uint256(1337)}();
     }
+
+    function testItCanReportGas() external {
+        ctx.startGasReport("test");
+        for (uint256 i = 0; i < 5; i++) {
+            new MockTarget();
+        }
+        ctx.endGasReport();
```

Comment:
I'd be curious to see a similar measurement done to see how they compare, e.g.

```solidity
uint256 startGas = gasleft();
for (uint256 i = 0; i < 5; i++) {
    new MockTarget();
}
println("gas used:", startGas - gasleft());
```

Reply:
sorry for the delay, we are in the middle of an audit right now, but will definitely look into this next week